File: Support Material/Demofiles/M12/VB/CustomerManagement/App_Code/Country.vb
Imports Microsoft.VisualBasic
Imports System.ComponentModel.DataAnnotations

Namespace CustomerManagementModel
    <ScaffoldTable(True)>
    Public Class CountryMetadata
        <ScaffoldColumn(False)>
        Public Property ID As Object
    End Class

    <MetadataType(GetType(CountryMetadata))>
    Partial Public Class Country

    End Class
End Namespace

File: Support Material/Demofiles/M15/CS/CustomerManagement/App_Code/Country.cs
using System.ComponentModel.DataAnnotations;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace CustomerManagementModel
{
    [ScaffoldTable(true)]
    public class CountryMetadata
    {
        [ScaffoldColumn(false)]
        public object ID;
    }

    /// <summary>
    /// Summary description for Country
    /// </summary>
    [MetadataType(typeof(CountryMetadata))]
    public partial class Country
    {
    }
}